pkg/database/migrations.go

```diff
@@ -2452,6 +2452,37 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 				)
 			},
 		},
+		{
+			ID: "00115-AddVerificationCodeConstraints",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx, `
+					ALTER SEQUENCE verification_codes_id_seq AS BIGINT;
```

Comment:
any concerns w/ these taking a long time to run?

Reply:
this could a long table lock over the verification_codes table.

Reply:
discussed & benchmarked offline - all good.

Reply:
Upgrading column types is constant (int -> bigint and varchar -> text), so I'm not concerned there.

Adding the FK constraints will lock for a full table scan for each foreign key. I ran this migration on local dev against a seed run with stats (which created 120k verification codes) and it took ~0.15s.